test/component/pages/pages.scala

```diff
@@ -99,11 +99,6 @@ object AddApplicationPage extends FormPage {
   override val url: String = s"${Env.host}/developer/applications/add"
 }
 
-object AddApplicationSuccessPage extends FormPage {
-  override val pageHeading = "Application added"
-  override val url: String = s"${Env.host}/developer/applications/add"
-}
-
 case object AccountDeletionConfirmationPage extends FormPage {
```

Comment:
What about trying this rather than removing AddApplicationSuccessPage:

`
object AddApplicationSuccessPage extends FormPage {
  override val pageHeading = "You added"
  override val url: String = s"${Env.host}/developer/applications/add"
  override def isCurrentPage: Boolean = find(tagName("h1")).fold(false)({
    e => Logger.info(s"HEADING: ${e.text}")
      e.text.startsWith(pageHeading)
  })
}
`